Cargo.toml

```diff
@@ -111,7 +111,7 @@ rolldown_utils                          = { version = "0.1.0", path = "./crates/
 anyhow              = "1.0.86"
 append-only-vec     = { version = "0.1.5" }
 arcstr              = "1.2.0"
-ariadne             = "0.5.0"
+ariadne             = { version = "0.5.0", git = "https://github.com/zesterer/ariadne", rev = "c9f858f18d30983f48ef8c726d36ec439b6000eb" }
```

Comment:
this blocks us to publish rolldown to creates.io

Reply:
I will create a new issue to track this, I will wait the author of `ariadne` to review the pr for a few days, if he don't response or reject the pr, we will maintain our fork.